openedx/features/course_duration_limits/tests/test_course_expiration.py

```diff
@@ -337,7 +337,7 @@ def test_no_banner_when_masquerading_as_staff(self, role_factory, mock_get_cours
             expired_staff = role_factory.create(password=TEST_PASSWORD, course_key=self.course.id)
 
         ScheduleFactory(
-            start=self.THREE_YEARS_AGO,
+            start_date=self.THREE_YEARS_AGO,
```

Comment:
unfortunately, I think this should also have been merged in the previous PR (https://github.com/edx/edx-platform/pull/22975).  I thought I already scanned for these changes, but I must have missed it?  Anyway, it's only a test, so it isn't urgent but I will find some time to correct this and run this test on my local devstack.

Reply:
>I will find some time to correct this and run this test on my local devstack.

I did just test edx-platform master with and without this change, and it succeeds.